CoffeeChat/Models/SimpleOnboardingCell.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  SimpleOnboardingCell.swift
+//  CoffeeChat
+//
+//  Created by Lucy Xu on 10/1/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import Foundation
+
+enum SimpleOnboardingCellType { case interest, normal }
+
+struct SimpleOnboardingCell {
```

Comment:
not sure if I missed something, but is this struct only used for the `InterestsGroupsViewController`, and not the other VCs that use the `SimpleOnboardingTableViewCell`? if that's the case, I feel like a different struct name that's specific to the interests/groups would be better, or refactoring so that all the VCs that use `SimpleOnboardingTableViewCell` use this model